xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp

```diff
@@ -351,6 +351,9 @@ bool CContext::CreateContext()
     m_atiWorkaround = true;
   }
 
+  // Sets high priority process for smooth playback in all circumstances
+  SetPriorityClass(GetCurrentProcess(), HIGH_PRIORITY_CLASS);
```

Comment:
you have to revert priority back after playback is stopped.

Reply:
I agree. It is done.